### PR TITLE
Create new_features.txt

### PR DESCRIPTION
**Pull request by Nellyougo**
I added new file with a new features, which we negotiated with our client!

### CODE REVIEW

File: new_features.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+We add new features for iur site:
+1. login and password
+2. main photo (car photo)
+3. Add name of our company in the centre of the cite page
```
Comment (Nellyougo) [inline new_features.txt:4]: Add new line after line 4 please

Comment (Nellyougo) [reply new_features.txt:4]: @Nellyougo I added new line! Please check it! %)

Comment (AlexPostID) [reply new_features.txt:4]: @Nellyougo hahaha

**Review by AlexPostID — APPROVED** [review]
All good, ready to merge

**Pull request merged**